aequilibrae/paths/basic_path_finding.pyx

```diff
@@ -14,7 +14,7 @@
  Licence:     See LICENSE.TXT
  -----------------------------------------------------------------------------------------------------------
 Original Algorithm for Shortest path (Dijkstra with a Fibonacci heap) was written by Jake Vanderplas <vanderplas@astro.washington.edu> under license: BSD, (C) 2012
```

Comment:
probably need to change this.

Reply:
Good catch, I've updated the other banner as well